src/pages/ProductList/Detail/ProductCartSide.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const ProductCartSide = props => {
+  const { name, price } = props;
+  const [count, setCount] = useState(1);
+  const navigate = useNavigate();
+
+  const onCart = () => {
+    //TODO:장바구니 담는 로직(fetch) api 입력
+    fetch('http://10.58.52.128:3000/carts', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: name,
+        quantity: count,
+        price: price,
+      }),
+    })
+      .then(response => response.json())
+      .then(data => {
+        if (data.message === 'SUCCESS') {
+          if (
+            window.confirm(
+              '장바구니에 담았습니다. 장바구니로 이동하시겠습니까?'
+            )
+          ) {
+            navigate('/cart');
+          }
+        } else {
+          alert('오류가 발생했습니다!');
+        }
+      });
+  };
+
+  const plusCount = () => {
+    setCount(prev => prev + 1);
+  };
+
+  const minusCount = () => {
+    if (setCount === 1) {
+      alert('최소 주문수량은 1개 입니다');
+    } else {
+      setCount(prev => prev - 1);
+    }
+  };
```

Comment:
```suggestion
  const minusCount = () => {
    if (setCount === 1) {
      alert('최소 주문수량은 1개 입니다'
      return;
    } else {
      setCount(prev => prev - 1);
    }
  };
```
alert이후로 이 함수가 더이상 동작하지 않기때문에 return을 적어주는게 좋겠습니다.